src/views/network/view_network.cpp

```diff
@@ -81,18 +81,24 @@ namespace big
 
 	void render_session_switcher()
 	{
+		static int selected_region_index     = -1;
+        std::string region_str = (selected_region_index == -1) ? "REGIONS"_T.data() : regions[*g_pointers->m_gta.m_region_code].name;
+
 		ImGui::BeginGroup();
 		components::sub_title("SESSION_SWITCHER"_T);
 		if (ImGui::BeginListBox("###session_switch", get_listbox_dimensions()))
 		{
-			if (ImGui::BeginCombo("##regionswitcher", "REGIONS"_T.data()))
+			if (ImGui::BeginCombo("##regionswitcher", region_str.c_str()))
 			{
 				for (const auto& region_type : regions)
 				{
 					components::selectable(region_type.name, *g_pointers->m_gta.m_region_code == region_type.id, [&region_type] {
 						*g_pointers->m_gta.m_region_code = region_type.id;
 					});
 				}
+
+				selected_region_index = *g_pointers->m_gta.m_region_code;
```

Comment:
You are not compensating for the user not selecting a region here.

Reply:
You're right. It was causing crashes for me in single player. Updating right now.